packages/web/src/app/web-player/WebPlayer.jsx

```diff
@@ -656,25 +665,38 @@ class WebPlayer extends Component {
                     />
                   )}
                 />
-
                 <Route
-                  path={SEARCH_CATEGORY_PAGE}
-                  render={(props) =>
-                    isSearchV2Enabled ? (
-                      <SearchPageV2 />
-                    ) : (
-                      <SearchPage
-                        {...props}
-                        scrollToTop={this.scrollToTop}
-                        containerRef={this.props.mainContentRef.current}
-                      />
-                    )
-                  }
+                  exact
```

Comment:
Ah ok so we're not toggling the routing behind a feature flag, I think that's fine

Reply:
yea the redirecting seems like a good way to have have the branching logic everywhere 👍